modules/adb-exfiltration-protection/main.tf

```diff
@@ -44,23 +31,3 @@ resource "azurerm_resource_group" "this" {
   location = local.location
   tags     = local.tags
 }
-
-output "arm_client_id" {
-  value = data.azurerm_client_config.current.client_id
-}
-
-output "arm_subscription_id" {
-  value = data.azurerm_client_config.current.subscription_id
-}
-
-output "arm_tenant_id" {
-  value = data.azurerm_client_config.current.tenant_id
-}
-
-output "azure_region" {
-  value = local.location
-}
-
-output "resource_group" {
-  value = azurerm_resource_group.this.name
-}
```

Comment:
We need to be careful with the removal of outputs - let's keep them for compatibility with existing modules, but add "deprecated" into description, and then remove when we'll cut a major release

https://github.com/hashicorp/terraform/pull/36016

Reply:
I made the changes recommended and will push once tested. Btw, I think the module for this example is too coarse, it may benefit from being broken down into smaller ones for re-use at some point